spanner/ddl/ast.go

```diff
@@ -211,7 +287,10 @@ func (ct CreateTable) PrintCreateTable(config Config) string {
 	if ct.Parent != "" {
 		interleave = ",\nINTERLEAVE IN PARENT " + config.quote(ct.Parent)
 	}
-	return fmt.Sprintf("%sCREATE TABLE %s (%s\n) PRIMARY KEY (%s)%s", tableComment, config.quote(ct.Name), cols, strings.Join(keys, ", "), interleave)
+	if config.TargetDb == "experimental_postgres" {
```

Comment:
Use TARGET_EXPERIMENTAL_POSTGRES constant?

Reply:
We couldn't use the constant before because of circular dependency. Have now moved all constants to a top-level common package and now have updated to use constant as well.